test/tests/selftest/pbs_default_timeout.py

```diff
@@ -55,8 +55,9 @@ def test_default_timeout(self):
             self.logger.info(mssg)
             time.sleep(MINIMUM_TESTCASE_TIMEOUT/2)
         except TimeOut, e:
-            mssg = 'Timed out after 600 second'
-            self.assertEqual(mssg, e.message)
+            mssg = 'Timed out after %s second' % MINIMUM_TESTCASE_TIMEOUT
+            err_mssg = 'failed due to mismatch in expected message'
```

Comment:
this log message doesn't really explain the main reason why the test failed. It might be more useful to say something like "the test timed out for an incorrect timeout duration". If you agree, then please make change the log message in the rest of the code as well.